src/index.html

```diff
@@ -1,17 +1,142 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta 
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="#" method="post">
+      <fieldset class="form-field">
+        <legend>
+          Personal information
+        </legend>
+        <label class="blok">
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="blok">
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="blok">
+          How old are You?
+          <input 
+            type="number" 
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="blok">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+            placeholder="dd.mm.yyyy"
+          >
+        </label>
+        <label class="blok">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="blok">
+            E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
```

Comment:
text should be on the same level as an input in case they are children of label. Apply for other cases as well
```suggestion
        <label class="blok">
          E-mail:
          <input 
            type="email" 
            name="email" 
            placeholder="email@example.com"
          >
        </label>
```